frontend/src/components/applyForm/FieldErrors.tsx

```diff
@@ -0,0 +1,29 @@
+import { ErrorMessage } from "@trussworks/react-uswds";
+
+export const FieldErrors = ({
+  fieldName,
+  rawErrors,
+}: {
+  fieldName: string;
+  rawErrors: string[] | undefined;
+}): React.ReactNode | null => {
+  if (rawErrors) {
+    if (rawErrors.length > 1) {
+      return (
+        <ErrorMessage id={`error-for-${fieldName}`}>
```

Comment:
This was simplified to only handle array of strings from the field widgets. The multiField widgets currently have their own logic. This should be simplified at some point.